iothub/device/src/Authentication/HsmAuthentication/HttpHsmSignatureProvider.cs

```diff
@@ -27,29 +28,17 @@ internal class HttpHsmSignatureProvider : ISignatureProvider
 
         public HttpHsmSignatureProvider(string providerUri, string apiVersion)
```

Comment:
Can we make any public an internal (those not inheriting from ISignatureProvider), like this ctor?